packages/sil-governance/src/routes/create-proposal/components/FormOptions/DistributeFunds/components/MembersTable.tsx

```diff
@@ -0,0 +1,36 @@
+import { Table, TableBody, TableCell, TableHead, TableRow } from "@material-ui/core";
+import { Typography } from "medulas-react-components";
+import React from "react";
+
+import { Recipient } from "..";
+
+interface Props {
+  readonly recipients: readonly Recipient[];
+}
+
+const MembersTable = ({ recipients }: Props): JSX.Element => {
+  return (
```

Comment:
You can simplify this code by removing `return` and curly braces from the component function.